### DISCUSSION TITLE
CellRank2 release date?

### DISCUSSION THREAD

**Discussion by YitengDang** [theislab/cellrank#961] [General]
Hi! I'm aware that you are working on CellRank2, which I saw presented at a conference recently. Do you have a rough indication of when this will be released? I'm trying to estimate whether I would be able to  squeeze in an analysis for a paper I'm working on using the new package.

**Comment by Marius1311** [comment]
We released CellRank 2 in July already, see my twitter post here: https://twitter.com/MariusLange8/status/1683505450497523714

This also includes a link to @WeilerP's post, as well as a link to the preprint.
